src/Welcome.js

```diff
@@ -1,77 +1,39 @@
-      [target.name]: target.value
-    });
-  }
-
-  handleSubmit(event) {
-    event.preventDefault();
-    this.setState({ patients: [] });
-    FHIRWrap.loadPatients(this.state.fhirServer, this.state.decedentName).then((patients) => {
-      this.setState({ patients });
-    });
-  }
-
-  handlePatientClick(event, data) {
-    event.preventDefault();
-    const patient = this.state.patients.find((patient) => patient.id === data.id);
-    FHIRWrap.loadResources(this.state.fhirServer, patient.id).then(([conditions, medications, procedures, observations]) => {
-      this.props.setResources(conditions, medications, procedures, observations);
-    });
-    this.props.setPatient(patient);
-    this.props.gotoStep('Pronouncing');
+  handleMenuClick(event, data) {
+    this.setState({ selectedMenuItem: data.name });
   }
 
   render() {
-
-    const patientLink = (patient) => {
-      return <Card key={patient.id} onClick={this.handlePatientClick} id={patient.id} header={patient.name} />;
-    };
-
-    const patientLinks = (patients) => {
-      if (!patients) {
-        return null;
-      } else if (patients.length === 0) {
-        return <Loader active inline='centered' content='Loading' />;
-      } else {
-        return <Card.Group className='patients'>{patients.map(patientLink)}</Card.Group>;
-      }
-    };
-
+    const { selectedMenuItem } = this.state;
     return (
       <div>
         <h2>Welcome</h2>
-        <p>This prototype application was developed as a collaboration between the <a href="http://miblab.bme.gatech.edu">Wang Lab</a> at Georgia Tech's Wallace H. Coulter Department of Biomedical Engineering and the Centers for Disease Control.</p>
-        <p>The purpose of this application is to provide visualization, context, and decision support at the point of a patient's death, with the aim of improving the timeliness, accuracy, and completeness of mortality reporting.</p>
-        <h2>Search for Decedent Record</h2>
-        <h3>Specify FHIR server and patient name to search for</h3>
-        <Form onSubmit={this.handleSubmit}>
-          <Form.Field>
-            <label>FHIR server:</label>
-            <input type="text" name="fhirServer" value={this.state.fhirServer} onChange={this.handleChange} />
-          </Form.Field>
-          <Form.Field>
-            <label>Decedent name:</label>
-            <input type="text" name="decedentName" value={this.state.decedentName} onChange={this.handleChange} />
-          </Form.Field>
-          <Button primary type="submit">Search</Button>
-        </Form>
-        {patientLinks(this.state.patients)}
+        <p>The purpose of this application is to provide visualization, context, and decision support at the point of a patient's death, with the aim of improving the timeliness, accuracy, and completeness of mortality reporting. Source code and documentation can be <a href="https://github.com/nightingaleproject/fhir-death-refactor">found on GitHub</a>.</p>
+        <Menu pointing>
```

Comment:
FYI: This annoying red highlighting is caused by Github's syntax highlighting not liking JSX code. For future projects, it's better to name these files `.jsx` instead of `.js` to not see this.